api/v1/storagecluster_types.go

```diff
@@ -85,6 +85,12 @@ type StorageClusterSpec struct {
 	// AllowRemoteStorageConsumers Indicates that the OCS cluster should deploy the needed
 	// components to enable connections from remote consumers.
 	AllowRemoteStorageConsumers bool `json:"allowRemoteStorageConsumers,omitempty"`
+
+	// ProviderAPIServerServiceType Indicates the ServiceType for OCS Provider API Server Service
+	// Supported Values are NodePort or LoadBalancer. The default ServiceType is NodePort if the value is empty
+	// This will only be used when AllowRemoteStorageConsumers is set to true
```

Comment:
```suggestion
	// ProviderAPIServerServiceType Indicates the ServiceType for OCS Provider API Server Service.
	// The supported Values are NodePort or LoadBalancer. The default ServiceType is NodePort if the value is empty.
	// This will only be used when AllowRemoteStorageConsumers is set to true
```